### ISSUE TITLE
Question: Is it possible to import a pre-tetrahedralized mesh?

### ISSUE THREAD

**Issue by crystalthoughts** [polyfem/polyfem#331]
I see there is no listing in the JSON schema for telling your library whether or not to generate interior geometry.
Being able to have fine grained control over the input mesh would be a great addition, if not already possible.

**Comment by teseoch** [comment]
Yes. PolyFEM expects a volumetric mesh as input and can directly load pre-tetrahedralized meshes such as `.msh` or `.mesh`. PolyFEM itself does not generate the tetrahedralization/interior mesh. Closing as answered.
